components/Rating/Rating.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ratings</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="./Rating.css" />
+  </head>
+  <body>
+    <h3>Fixed ratings</h3>
+    <div>
+      <i class="fas fa-star rating rating-fixed"></i>
+      <i class="fas fa-star rating rating-fixed"></i>
+      <i class="fas fa-star rating rating-fixed"></i>
+      <i class="far fa-star rating rating-fixed"></i>
+      <i class="far fa-star rating rating-fixed"></i>
+    </div>
+    <h1>Dynamic ratings</h1>
```

Comment:
You can avoid using "h1" tag here, as it's just another heading. "h1" tags concisely describe the overall purpose of the content. (Just a best practice).

Reply:
Thank u Manoj ! i didn't know that point about H1 tags